request.js

```diff
@@ -1413,6 +1413,28 @@ Request.prototype.jar = function (jar) {
   return self
 }
 
+Request.prototype.cloneJar = function (cookieJar) {
+  var self = this
+  var newJar, cookieString, cookieStore, domains, domainPaths, cookieUrl
+
+  newJar = self.jar()
```

Comment:
I don't think this works. I'm running your code on my end and Im not getting a Jar returned from this statement.
Im getting back the request instance. Seems like Request.prototype.jar takes in a jar as well.
Perhaps you meant to say `cookies.jar()` ?